card/db/migrate_core_cards/20150510031118_fix_skin_images.rb

```diff
@@ -0,0 +1,13 @@
+# -*- encoding : utf-8 -*-
+
+class FixSkinImages < Card::CoreMigration
+  def up
+
+    # This is needed because the bootswatch_themes migration removed codenames.
+    # They were no longer needed for style handling but are still needed for images
+    %w{bootstrap_default cerulean cosmo cyborg darkly flatly journal lumen paper readable sandstone simplex slate spacelab superhero united yeti }.each do |theme_name|
+      theme_name = "#{theme_name}_skin"
+      Card.fetch( theme_name ).update_attributes! :codename=>theme_name
+    end
+  end
```

Comment:
comment above mostly explains it.

we may be able to get rid of this migration before putting out the gem, but we have to have it to fix the broken images on cloudstore